### ISSUE TITLE
ABC not transposing (6.030_032)

### ISSUE THREAD

**Issue by gwyndaf** [ChordPro/chordpro#325]
Also experiencing same issue with 6.030_035.

Test sample:
```
{title: ABC test}
{key: F}
{transpose: +2}
#
{start_of_abc}
X:1
M:4/4
L:1/16
K:F
cAFc AFcA cAFc AFcA | cAFc AFcA _e_BG_e _BG_e_B | cAFc AFcA d_BFd _BFd_B |

{end_of_abc}
```
I'd expect this to create a score in the key of G (F +2 semitones). Output is generated, but it's still in F.

`chordpro-git -X --def debug.abc=2 abc_test.cho` gives me debug output of
```
ABC: abc2svg (tool = <undef>)
%%pagewidth 515px
%%topspace 0
%%titlespace 0
%%musicspace 0
%%composerspace 0
%%infospace 0
%%textspace 0
%%leftmargin 0cm
%%rightmargin 0cm
%%stretchstaff 1
%%stretchlast 0
%%trimsvg 1
%%staffsep 0
%%textfont pdf.fonts.text
%%gchordfont pdf.fonts.chord
X:1
M:4/4
L:1/16
K:F
cAFc AFcA cAFc AFcA | cAFc AFcA _e_BG_e _BG_e_B | cAFc AFcA d_BFd _BFd_B |

+ /home/alun/Scripts/abc2svg /tmp/554kdc064U/tmp1.abc
SVG: 91 lines (raw)
SVG: 72 lines (34469 bytes)
```

I'm assuming (!) that `abc2svg` does the transposing work, in response to a directive from ChordPro, but I can't see anything in the `%%` preamble that seems to do that.

`ChordPro/Delegate/ABC.pm` mentions `%%transpose` at line 168. I don't understand Perl, but at guessing that might be relevant. Maybe I should expect to see a `%%transpose` directive in the ABC preamble of debug output?

**Comment by sciurius** [comment]
Long story short: Previously, ABC was rendered in situ. Now it is a two stage process and the transpose setting was not yet carried over from the first stage to the second.

Fixed in 6.030_037.

**Comment by gwyndaf** [comment]
Great. Looking good again with _037  :)
